portal/src/examples/AccordionExample.tsx

```diff
@@ -10,3 +18,17 @@ export const accordionExample = `
         interest and sympathy.
     </AccordionItem>
 </Accordion>`;
+
+const exampleImport = `import { Accordion, AccordionItem } from "@fremtind/jkl-accordion-react";
+import "@fremtind/jkl-accordion/accordion.min.css";`;
+
+const AccordionExample = () => (
+    <Example
+        exampleComponents={{ Accordion, AccordionItem }}
+        exampleCode={example}
+        types={[{ name: "Accordion", type: accordionType }, { name: "AccordionItem", type: accordionItemType }]}
+        exampleImport={exampleImport}
+    />
+);
+
+export default AccordionExample;
```

Comment:
Are we starting to over-complicate the examples? This probably needs some explanation for someone writing their first component example. Maybe clarify that examples only need the sample code string to work?

Reply:
Well, yes. However, it's not more complicated now than it used to be, it's just that everything is located in a single file, insted of beeing spread out into two.

Reply:
But yes, as the functionallity grows, the learing curve also grows. And we should document how to use it. I do think its still quite easy to grasp, and its just plain js, no magic.

Reply:
Yeah, I don't think we've gone too far, but we're at a point where we should probably keep it in mind when/if we do further enhancements. The bar for documenting should be **really** low 😅